### PR TITLE
System.Decimal F# snippets

### PR DESCRIPTION
**Pull request by albert-du**
Adds F# snippets for Decimal

### CODE REVIEW

**Comment by albert-du** [comment]
@dsyme Could you review?

**Comment by BillWagner** [comment]
@albert-du @dsyme 

I read through the code, and this looks good to me. Once you agree, I'll :shipit:

File: samples/snippets/fsharp/VS_Snippets_CLR/Decimal Example/FS/fs.fsproj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (gewarren) [inline samples/snippets/fsharp/VS_Snippets_CLR/Decimal Example/FS/fs.fsproj:1]: Can you move all the snippet files up a level? We removed the top-level "samples" folder recently.

Comment (albert-du) [reply samples/snippets/fsharp/VS_Snippets_CLR/Decimal Example/FS/fs.fsproj]: 👍

**Review by dsyme — APPROVED** [review]

**Comment by albert-du** [comment]
@BillWagner this is ready

**Pull request merged**